<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>twnltClass</class>
 <widget class="QMainWindow" name="twnltClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1676</width>
    <height>820</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1676</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>公交转乘</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/twnlt/华中科技大学周边地图.bmp);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1676</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
    <addaction name="action_find"/>
   </widget>
   <widget class="QMenu" name="menu_Q">
    <property name="title">
     <string>查询(Q)</string>
    </property>
    <addaction name="action_zszc"/>
    <addaction name="action_zdjl"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_Q"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionduruditu">
   <property name="text">
    <string>duruditu</string>
   </property>
  </action>
  <action name="action_open">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>读入数据</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_zszc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>最少转乘</string>
   </property>
  </action>
  <action name="action_zdjl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>最短距离</string>
   </property>
  </action>
  <action name="action_find">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>模糊匹配</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="twnlt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_zszc</sender>
   <signal>triggered(bool)</signal>
   <receiver>twnltClass</receiver>
   <slot>zszc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_zdjl</sender>
   <signal>triggered(bool)</signal>
   <receiver>twnltClass</receiver>
   <slot>zdjl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_open</sender>
   <signal>triggered(bool)</signal>
   <receiver>twnltClass</receiver>
   <slot>loadmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered(bool)</signal>
   <receiver>twnltClass</receiver>
   <slot>closewnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_find</sender>
   <signal>triggered(bool)</signal>
   <receiver>twnltClass</receiver>
   <slot>findinput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>zszc()</slot>
  <slot>zdjl()</slot>
  <slot>loadmap()</slot>
  <slot>closewnd()</slot>
  <slot>findinput()</slot>
 </slots>
</ui>
